.github/workflows/ci.yml

```diff
@@ -15,23 +15,25 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Use Node.js 14
       uses: actions/setup-node@v1
       with:
-        node-version: '14.1.0'
+        node-version: '14'
     - run: npm install
     - run: npm install --prefix ui
+
     - run: npm run lint
     - run: npm run lint --prefix ui
+    - run: npm run build --prefix ui
```

Comment:
We need to run `npm run build --prefix ui` otherwise the `ui-bundle.zip` won't be generated.